app/hub/plugins/compatibility.md

```diff
@@ -98,5 +96,171 @@ see [{{site.ee_product_name}} for Kubernetes Deployment Options](/gateway/latest
 
 ### Deployment
 
-[Deployment plugins](/hub) are not bundled with any version of Konnect, and are
+[Deployment plugins](/hub/#deployment) are not bundled with any version of {{site.base_gateway}} or {{site.konnect_short_name}}, and are
 simply tools to help you deploy {{ site.base_gateway }} in various environments.
+
+## Protocols
+
+{{site.base_gateway}} and {{site.konnect_short_name}} plugins are compatible with the following protocols:
+
+{% assign hub = site.data.ssg_hub %}
+{% assign kong_extns = hub | where: "publisher", "Kong Inc." %}
+
+<table class="table-sticky">
+  <thead>
+      <th>Plugin</th>
+      <th><code>http</code></th>
+      <th><code>https</code></th>
+      <th><code>grpc</code></th>
+      <th><code>grpcs</code></th>
+      <th><code>tcp</code></th>
+      <th><code>tls</code></th>
+      <th><code>tls_ passthrough</code></th>
+      <th><code>udp</code></th>
+      <th><code>ws</code></th>
+      <th><code>wss</code></th>
+  </thead>
+  <tbody>
+  {% for extn in kong_extns %}
+    <tr>
+      <td>
+      <a href="{{extn.url}}">{{ extn.name }}</a>
+      </td>
+      {% if extn.params.protocols == null %}
+        <td>--</td>
```

Comment:
I think this is what is displaying for Vault Auth in the table? If it is, just a nitpick that the spacing looks a bit off with the emoji cross and checkmark, not sure if that's something could be fixed. Also, I think someone in #docs mentioned that they were confused about what "--" meant. So maybe something like N/A would work instead?

Reply:
Hmm. Good call. Will do N/A.